Game of Game of Life.py

```diff
@@ -1,18 +1,7 @@
 ### a,b refers to cell (a,b) whereas x,y refers to pixel coordinates ###
 
 import pygame as pg, math as m, time as t, random as r
```

Comment:
Change to one import per line and don't do `import as`. That is used rarely and only for popular packages like numpy, and if you use it on every package the code is harder to read because you need to always keep in mind that the package was renamed. Read https://www.python.org/dev/peps/pep-0008/#imports for more

Reply:
done